tools/report_build.py

```diff
@@ -245,31 +286,49 @@ class RepoCall(GitHubV3Call):
     def run(self, variables={}):
         self._repo = self.github.get_repo(
             "{owner}/{name}".format(**self._vars(variables)))
+        return self
 
 
-class AddCommitComment(RepoCall):
+class CommentURLMixin():
+    @property
+    def url(self):
+        return self.data.html_url
+
+
+class AddCommitComment(RepoCall, CommentURLMixin):
     def run(self, variables={}):
         super().run(variables)
-        return (self._repo.get_commit(self._vars(variables)["sha"])
-                .create_comment(self._vars(variables)["comment"]["body"]))
+        self.data = (self._repo.get_commit(self._vars(variables)["sha"])
+                     .create_comment(self._vars(variables)["comment"]["body"]))
+        return self
 
 
-class UpdateCommitComment(RepoCall):
+class UpdateCommitComment(RepoCall, CommentURLMixin):
     def run(self, variables={}):
         super().run(variables)
-        return (self._repo.get_comment(
+        self.data = self._repo.get_comment(
+            self._vars(variables)["databaseId"])
+        (self._repo.get_comment(
```

Comment:
This seems forgotten, `edit` should just be called on `self.data`.

Reply:
Fixed.